.env

```diff
@@ -0,0 +1,9 @@
+DOCKERHUB_NAME=username
```

Comment:
This shouldn't be committed and push up to github. Make sure you add this to your .gitignore